segmentation/segmentation.py

```diff
@@ -0,0 +1,31 @@
+# segmentation.py
+# Script that loops through a folder and segments audio into 10 second snippets
+# or smaller
+
+from pathlib import Path  # For writing videos into the data folder
+from pydub import AudioSegment  # This also requires ffmpeg on the device
+import sys
+
+date_path = Path(Path.cwd().parent, "data", "reddit", sys.argv[1])
+
+# Pydub uses milleseconds as the units; 10 000 is 10 seconds
+ten_seconds = 10 * 1000
```

Comment:
I thinnnkkk we should do 30 seconds for now, and then see if that is too long later. I think 10 may not be long enough to get useful info

Reply:
I've [increased the duration to 30 seconds](https://github.com/myHaiven/data-collection/pull/4/commits/d8d339df7c59a3035bbdc8b3ab9e2148eb6f86c7) for the audio segmentation script.